R/mod_quality_overview.R

```diff
@@ -227,6 +228,23 @@ mod_quality_overview_server <- function(input,
         sort(decreasing = T)
     )
   })
+
+  output$appInfo <- shiny::renderUI({
+    shiny::actionButton(
+      inputId = ns("app_info"),
+      label = "",
+      icon = shiny::icon("info")
+    )
+  })
+  shiny::observeEvent(input$app_info, {
+    shinyalert::shinyalert(title = config$app_text$info$title,
+                           text = qmongr::version_info(),
+                           type = "",
+                           closeOnEsc = TRUE, closeOnClickOutside = TRUE,
+                           html = TRUE,
+                           confirmButtonText = qmongr::no_opt_out_ok())
+  })
```

Comment:
Is it possible to test this part of code?

Reply:
I'll try

Reply:
superficially, at least